app/src/main/java/com/brainwallet/data/repository/LtcRepository.kt

```diff
@@ -51,6 +54,10 @@ interface LtcRepository {
             }.getOrElse { Fee.Default }
         }
 
+        override suspend fun fetchLimits(baseCurrencyCode: String): MoonpayCurrencyLimit =
```

Comment:
Is this working now @andhikayuana ?

Reply:
some baseCurrency works and some errors got errors
https://github.com/gruntsoftware/internal/issues/115#issuecomment-2847821236